examples/generative/vq_vae.py

```diff
@@ -357,7 +366,23 @@ def show_subplot(original, reconstructed):
 print(f"Input shape of the PixelCNN: {pixelcnn_input_shape}")
 
 """
-Don't worry about the input shape. It'll become clear in the following sections.
```

Comment:
This remark confused the hell out of me for quite a while. I took some time to write out what this shape exactly means as clearly and concisely as I possibly could.

Reply:
:+1: updated in https://github.com/keras-team/keras-io/pull/938/commits/026df677bf739d5dbb7a4134aaf57cfa54d497ab